Base/QTGUI/qSlicerModuleFinderDialog.h

```diff
@@ -0,0 +1,64 @@
+/*==============================================================================
+
+  Program: 3D Slicer
+
+  Copyright (c) Kitware Inc.
+
+  See COPYRIGHT.txt
+  or http://www.slicer.org/copyright/copyright.txt for details.
+
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.
+
+==============================================================================*/
+
+#ifndef __qSlicerModuleFinderDialog_h
+#define __qSlicerModuleFinderDialog_h
+
+// CTK includes
+#include <ctkPimpl.h>
+
+// Slicer includes
+#include "QDialog.h"
```

Comment:
`QDialog.h` -> `QDialog` 

Move to `// Qt includes` sections

Reply:
ok